docs/reference/index-modules/similarity.asciidoc

```diff
@@ -494,36 +504,48 @@ it is <<indices-create-index,created>>:
 
 [source,js]
 --------------------------------------------------
-PUT /my_index
+PUT index
 {
   "settings": {
     "index": {
       "similarity": {
         "default": {
-          "type": "boolean"
+          "type": "classic"
         }
       }
     }
   }
 }
 --------------------------------------------------
+// CONSOLE
 
 If you want to change the default similarity after creating the index
-you must <<indices-open-close,close>> your index, send the follwing
+you must <<indices-open-close,close>> your index, send the following
 request and <<indices-open-close,open>> it again afterwards:
 
+////////////////////
+
 [source,js]
 --------------------------------------------------
-PUT /my_index/_settings
+POST index/_close
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+////////////////////
+
+[source,js]
+--------------------------------------------------
+PUT index/_settings
```

Comment:
Maybe just put the `POST /index/_close` request in the snippet so the users sees it. We tell them to close but it is a fine reminder of how.